docs/book/src/topics/gpu.md

```diff
@@ -0,0 +1,129 @@
+# GPU-enabled clusters
+
+## Overview
+
+With CAPZ you can create GPU-enabled Kubernetes clusters on Microsoft Azure.
+
+Before you begin, be aware that:
+
+- [Scheduling GPUs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/) is a Kubernetes beta feature
+- [NVIDIA GPUs](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu) are supported on Azure NC-series, NV-series, and NVv3-series VMs
+
+To deploy a cluster with support for NVIDIA GPU nodes, use the [nvidia-gpu flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/templates/cluster-template-nvidia-gpu.yaml).
+
+## An example GPU cluster
+
+Let's create a CAPZ cluster with an NVIDIA node and run a GPU-powered vector calculation.
+
+### Generate an nvidia-gpu cluster template
+
+Use the `clusterctl config cluster` command to generate a manifest that defines your GPU-enabled
+workload cluster.
+
+Remember to use the `nvidia-gpu` flavor with N-series nodes.
```

Comment:
let's also add the new flavor to https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/master/templates/flavors#from-tilt-config

Reply:
stretch: this is a great opportunity to demonstrate variable override by setting `AZURE_NODE_MACHINE_TYPE` for the gpu flavor https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/master/templates/flavors#defining-variable-overrides

Reply:
Ooh good call, yes let's show how to do it in tilt-settings.json.

Reply:
Done, let me know what you think.